models/models.go

```diff
@@ -73,56 +57,40 @@ type QueryParams struct {
 	Resource    string
 }
 
-// TODO: Seperate database struct and request struct
 type ExportPayload struct {
-	ID          uuid.UUID      `gorm:"type:uuid;primarykey" json:"id"`
-	CreatedAt   time.Time      `gorm:"autoCreateTime" json:"created"`
-	UpdatedAt   time.Time      `gorm:"autoUpdateTime" json:"-"`
-	CompletedAt *time.Time     `json:"completed,omitempty"`
-	Expires     *time.Time     `json:"expires,omitempty"`
-	RequestID   string         `json:"-"`
-	Name        string         `json:"name"`
-	Format      PayloadFormat  `gorm:"type:string" json:"format"`
-	Status      PayloadStatus  `gorm:"type:string" json:"status"`
-	Sources     datatypes.JSON `gorm:"type:json" json:"sources"`
-	S3Key       string         `json:"-"`
+	ID          uuid.UUID `gorm:"type:uuid;primarykey"`
```

Comment:
Why didn't it format this haha

Reply:
Yeah... strange
I think it might be a problem with my VSCode